lib/appsignal/utils.rb

```diff
@@ -14,12 +14,81 @@ def sanitize(params, only_top_level = false, key_sanitizer = nil)
     end
     extend ClassMethods
 
+    def self.data_generate(body)
+      Data.generate(body)
+    end
```

Comment:
Agree in general. I mirrored the existing Json implementation to make the change as small as possible.